mypy.ini

```diff
@@ -0,0 +1,11 @@
+[mypy]
```

Comment:
Is it possible to add these configurations to tox.ini, instead of creating a new file?

Reply:
yes! working on it

Reply:
@sfc-gh-jdu I don't think it's possible to move mypy.ini to tox.ini here since tox.ini does not support module-specific commands. For instance, the paths `snowflake.*`. `cloudpickle.*`, and `pandas.*` need to ignore missing imports but it's not possible to mention this in tox.ini without applying it to everything. So I will keep the mypy.ini file here

Reply:
got it